src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md

```diff
@@ -57,6 +57,27 @@ To ensure the stability of your customizations and prevent upgrades from overwri
 
 A `<font>` node is added to HTML `<head>` type for layout in `lib/internal/Magento/Framework/View/Layout/etc/head.xsd`. All resources added with `<font>` node will be downloaded with `preload` html attribute.
 
+## Using Google fonts
+
+Google fonts has vast majority of fonts library that is easy to use.
+
+Here is a simple explanation with an example on how to implement this in your custom theme.
+
+1. Go to <Vendor>/<theme>/Magento_Theme/layout/default_head_blocks.xml or create one if it doesn't exist.
+1. Choose your desired font from google fonts, copy the import code and add the <link> inside the <head> section as like in the example below.
+
+```xml
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <link rel="stylesheet" type="text/css" src="https://fonts.googleapis.com/css?family=Roboto+Condensed:400,400i,700,700i&amp;display=swap" src_type="url" />
```

Comment:
Hi @karthik-ziffity. Thank you for your contribution. I would not recommend adding Google Fonts as an external source for fonts. It's a well-known way of compromising storefront performance. Despite the fact that a sort of CDN is being used for  fonts.googleapis.com, Google still does not guarantee good performance for this approach. Instead of using Google APIs as a source, it's recommended to store fonts using a dedicated CDN service where all your static assets are being stored (so you have control over your static assets including performance perspective).

Reply:
Okay, Thanks. Should I close this PR then?